<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>igwStreamLinesRepresentationWidget</class>
 <widget class="QWidget" name="igwStreamLinesRepresentationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>属性:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="DataArrayComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>流线形状</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="StreamTypeComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>流线</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>流管</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>流带</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>流面</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>发射方向</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="DirectionComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>+</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>+/-</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>流线粗细</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="WidthSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::DefaultStepType</enum>
         </property>
         <property name="value">
          <number>1</number>
         </property>
         <property name="displayIntegerBase">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>流线长度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="LengthSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::DefaultStepType</enum>
         </property>
         <property name="value">
          <number>5000</number>
         </property>
         <property name="displayIntegerBase">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>种子点选取</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>发射于</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>平面</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>线</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>发射点数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="PlaneXAxisSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::DefaultStepType</enum>
          </property>
          <property name="value">
           <number>5</number>
          </property>
          <property name="displayIntegerBase">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="PlaneYAxisSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::DefaultStepType</enum>
          </property>
          <property name="value">
           <number>5</number>
          </property>
          <property name="displayIntegerBase">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>种子点数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="NumOfLineSeedsSpinBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::DefaultStepType</enum>
          </property>
          <property name="value">
           <number>50</number>
          </property>
          <property name="displayIntegerBase">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="igwPlaneParametersWidget" name="PlaneWidget"/>
        <widget class="igwLineParametersWidget" name="LineWidget"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>igwPlaneParametersWidget</class>
   <extends>QWidget</extends>
   <header>igwplaneparameterswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>igwLineParametersWidget</class>
   <extends>QWidget</extends>
   <header>igwlineparameterswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
